terraform/modules/spack/gitlab_webhooks.tf

```diff
@@ -16,3 +17,53 @@ resource "gitlab_project_hook" "job_webhook" {
 
 // TODO: Once https://gitlab.com/gitlab-org/terraform-provider-gitlab/-/issues/1350 is resolved the
 // gitlab_application_settings resource should be used to whitelist the domains in job_webhooks.
+
+
+data "gitlab_user" "spackbot" {
+  username = "spackbot"
```

Comment:
Does it make sense to use spackbot here, or something else? @AlmightyYakob @mvandenburgh